module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -371,7 +398,7 @@ namespace sys
     bool SystemManagerCommon::DestroyApplication(const std::string &name, Service *caller)
     {
         cpp_freertos::LockGuard lck(appDestroyMutex);
-        if (RequestServiceClose(name, caller)) {
+        if (RequestServiceClose(name, caller, 1000)) {
```

Comment:
This has been 5000 by default. Why shortening time so much?

Reply:
it was temporary change, will revert

Reply:
tx